clients/cloud/java/src/main/java/io/confluent/examples/clients/cloud/ProducerExample.java

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright 2015 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.confluent.examples.clients.cloud;
+
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.clients.producer.RecordMetadata;
+import org.apache.kafka.clients.producer.Callback;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.connect.json.JsonSerializer;
+
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.AdminClientConfig;
+import org.apache.kafka.clients.admin.NewTopic;
+
+import io.confluent.examples.clients.cloud.model.RecordJSON;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import org.apache.kafka.common.errors.TopicExistsException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Properties;
+import java.util.Collections;
+import java.util.Map;
+
+public class ProducerExample {
+
+  private static final ObjectMapper MAPPER = new ObjectMapper();
+
+  // Create topic in Confluent Cloud
+  public static void createTopic(final String topic,
+                          final int partitions,
+                          final int replication,
+                          final Properties cloudConfig) {
+      final NewTopic newTopic = new NewTopic(topic, partitions, (short) replication);
+      try (final AdminClient adminClient = AdminClient.create(cloudConfig)) {
+          adminClient.createTopics(Collections.singletonList(newTopic)).all().get();
+      } catch (final InterruptedException | ExecutionException e) {
+          // Ignore if TopicExistsException, which may be valid if topic exists
+          if (!(e.getCause() instanceof TopicExistsException)) {
+              throw new RuntimeException(e);
+          }
+      }
+  }
+
+  public static void main(String[] args) throws IOException {
+    if (args.length != 2) {
+      System.out.println("Please provide command line arguments: configPath topic");
+      System.exit(-1);
```

Comment:
exit values should be positive, 1 is typical for error .

Reply:
Changed to 1.

These are all errors resulting from copy/paste other code (from 2015, apparently)